.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module",
+    "project": "./tsconfig.json"
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:node/recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:import/typescript",
+    "plugin:prettier/recommended"
+  ],
+  "rules": {
+    "node/no-unsupported-features/es-syntax": [
+      "error",
+      {
+        "ignores": ["modules"]
+      }
+    ],
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "import/named": "off"
+  },
+  "settings": {
+    "import/parsers": { "@typescript-eslint/parser": [".ts"] },
+    "node": {
+      "convertPath": {
+        "src/**/*.ts": ["^src/(.+?)\\.ts$", "dist/$1.js"]
```

Comment:
let's the node plugin know that transpilation is happening, and the `dist` folder is constructed from the `src` folder